swagger/fs_wrapper.go

```diff
@@ -0,0 +1,95 @@
+package swagger
+
+import (
+	"bytes"
+	"html/template"
+	"io"
+	"io/fs"
+	"strings"
+	"time"
+)
+
+type fsWrapper struct {
+	fs           fs.FS
+	jsonFileName string
+}
+
+func FsWrapper(fs fs.FS, jsonFilePath string) *fsWrapper {
+	return &fsWrapper{
+		fs:           fs,
+		jsonFileName: jsonFilePath,
+	}
+}
+
+func (f *fsWrapper) Open(name string) (fs.File, error) {
+	if strings.HasSuffix(name, "index.html") {
```

Comment:
Itsn't it sufficient to compare with `index.html` ? Do we expect a file with a different prefix ?

Reply:
> You're right. However, I stil dislike that the user has to pass the `fizz.Engine()` result. We could instead provide a `fizz` method that would be called `ServeSwaggerUI(path, specPath string)` that would do the same under the hood without exposing the client to the underlying "machinery".

We had fizz method in the initial version but if the method is in the same package then assets are always part of the resulting binary even if UI is not used

Reply:
I'm open to suggestions though